packages/babel-plugin-proposal-async-generator-functions/test/fixtures/async-generators/transform-class-keys/babel.config.json

```
@@ -0,0 +1,7 @@
+{
+    "sourceType": "unambiguous",
+    "presets": ["@babel/preset-env"],
+    "plugins": [
+      "@babel/plugin-transform-runtime"
+    ]
```

We can simplify this:
```suggestion
    "plugins": ["proposal-async-generator-functions"]
```